spec/units-of-measure.md

```diff
@@ -155,105 +148,82 @@ After measures are parsed and checked, they are maintained in the following norm
 
-To check the equality of two measures, abbreviations are expanded to compare their normalized
-forms. However, abbreviations are not expanded for presentation. For example, consider the
-following definitions:
+To check the equality of two measures, abbreviations are expanded to compare their normalized forms. However, abbreviations are not expanded for presentation. For example, consider the following definitions:
 
 ```fsharp
 [<Measure>] type a
 [<Measure>] type b = a * a
 let x = 1<b> / 1<a>
 ```
-The inferred type is presented as `int<b/a>`, not `int<a>`. If a measure is equivalent to `1` , however, abbreviations
-are expanded to cancel each other and are presented without units:
+
+The inferred type is presented as `int<b/a>`, not `int<a>`. If a measure is equivalent to `1`, however, abbreviations are expanded to cancel each other and are presented without units:
 
 ```fsharp
 let y = 1<b> / 1<a a> // val y : int = 1
 ```
+
 ### Constraint Solving
 
-The mechanism described in [§14.5](inference-procedures.md#constraint-solving) is extended to support equational constraints between measure
-expressions. Such expressions arise from equations between parameterized types — that is, when
-`type<tyarg11 , ..., tyarg1n> = type<tyarg21, ..., tyarg2n>` is reduced to a series of constraints
-`tyarg1i = tyarg2i`. For the arguments that are measures, rather than types, the rules listed in [§9.3](units-of-measure.md#relations-on-measures)
-are applied to obtain primitive equations of the form `'U = measure-int` where `'U` is a measure
-variable and `measure-int` is a measure expression in internal form. The variable `'U` is then replaced
-by `measure-int` wherever else it occurs. For example, the equation `float<m^2/s^2> = float<'U^2>`
-would be reduced to the `constraint m^2/s^2 = 'U^2`, which would be further reduced to the primitive
-equation `'U = m/s`.
+The mechanism described in [§14.5](inference-procedures.md#constraint-solving) is extended to support equational constraints between measure expressions. Such expressions arise from equations between parameterized types — that is, when `type<tyarg11 , ..., tyarg1n> = type<tyarg21, ..., tyarg2n>` is reduced to a series of constraints `tyarg1i = tyarg2i`. For the arguments that are measures, rather than types, the rules listed in [§9.3](units-of-measure.md#relations-on-measures) are applied to obtain primitive equations of the form `'U = measure-int` where `'U` is a measure variable and `measure-int` is a measure expression in internal form. The variable `'U` is then replaced by `measure-int` wherever else it occurs. For example, the equation `float<m^2/s^2> = float<'U^2>` would be reduced to the `constraint m^2/s^2 = 'U^2`, which would be further reduced to the primitive equation `'U = m/s`.
```

Comment:
Should we not move this to §14.5 rather than keeping it here?

Reply:
Hmm... 🤔 On the one hand, yes. It's constraint solving; so keeping all that stuff together is a good idea. But on the other hand, keeping all the "units of measure" stuff in one place is convenient for grokking the features (without having to bounce around). I am genuinely torn.

Reply:
I agree. And since moving it will introduce chances for mistakes, it's perhaps best to keep it here and rather spend the time on the next RFC  :-)